demos/demo_runner/iot_demo_freertos.c

```diff
@@ -346,6 +359,13 @@ void runDemoTask( void * pArgument )
         IotLogError( "Failed to initialize the demo. exiting..." );
     }
 
+    #ifdef democonfigMEMORY_ANALYSIS
+        xAfterDemoHeapSize = demoMEMORY_ANALYSIS_MIN_EVER_HEAP_SIZE();
+        IotLogInfo( "Demo Memory Analysis Heap Total: %u Before: %u After: %u \r\n", configTOTAL_HEAP_SIZE, xBeforeDemoHeapSize, xAfterDemoHeapSize );
```

Comment:
Minor clarification -- "before" and "after" may confuse people. The return value of "minimum ever heap size" is 'the lowest amount of free heap space that has existed in system since the FreeRTOS application booted". This is not equivalent to "heap space available then and now".

Reply:
``` IotLogInfo( "Demo Memory Analysis Heap Total: %u Minimum heap size ever before running demo: %u after running demo: %u \r\n", configTOTAL_HEAP_SIZE, xBeforeDemoHeapSize, xAfterDemoHeapSize );```

Would this be more clear?

Reply:
Works, thank you!